src/marks/geo.js

```diff
@@ -23,6 +23,8 @@ export class Geo extends Mark {
       data,
       {
         geometry: {value: options.geometry, scale: "projection"},
+        x: {value: options.x, scale: "x", optional: true},
+        y: {value: options.y, scale: "y", optional: true},
```

Comment:
They are used by the tip mark, which _derives_ the x and y channels from the geo mark. That's visible when interacting with the londonCarAccessGeoCentroid example (but unfortunately not caught by the automated test).

Reply:
If I’m understanding correctly, you’re suggesting that the geo mark should be responsible for preparing the _x_ and _y_ channels for the **tip** option (and its derived tip mark) even though the geo mark doesn’t need these channels for itself.

This doesn’t feel like the right place to fix this problem. Either all marks should implicitly support _x_ and _y_ channels if declared under the expectation that maybe you’ll use the **tip** option and the derived tip mark will need them, or the geoCentroid transform should be responsible for declaring _x_ and _y_ as custom channels rather than simply passing them through as options.

Another possibility is that the geo mark should just support the **tip** option directly by implicitly invoking the geoCentroid transform (somehow?), or that the `derive` function for deriving a tip from an existing mark should similarly promote the _geometry_ channel into _x_ and _y_ channels, or that the tip mark and pointer transform should directly support reading the _geometry_ channel as an alternative to _x_ and _y_ channels.